include/mesh/mesh_smoother_vsmoother.h

```diff
@@ -65,8 +65,9 @@ class UnstructuredMesh;
  *
  * Notes:
  *
- * 1) The smoother supports tangled meshes. However, it cannot untangle all such
- * meshes. This will be investigated further in the future.
+ * 1) The smoother supports tangled meshes. However, not all untangling solves
+ * converge. Some will converge to a local optima, resulting in an untangled,
+ * but still suboptimal mesh that cannot be smoothed much further.
```

Comment:
Wait what now?  I thought our problem was that there were some meshes that we couldn't untangle.  If there are meshes that we can untangle but that don't converge to a global optimum afterward, doesn't that mean that not all solves on *untangled* meshes converge?

Reply:
I'll add another thought to my comment above.

In the case that a tangled mesh is untangled to a non-global optima, I think it is quite possible that it cannot be smoothed much further, especially if the smooth global optima cannot be reached without traveling through tangled mesh space.

Reply:
Well, after trying to find a case where we untangle a mesh, but smooth it to a local optima, I am prepared to eat my words. I couldn't find such a case occurring after I patched all the bugs. I'll update the wording in the documentation.